apps/rotm/views/confirmation.html

```diff
@@ -0,0 +1,6 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{>partials-confirmation-alert}}
+    {{#markdown}}confirmation{{/markdown}}
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
What's the logic behind extending this template and not using the default? https://github.com/UKHomeOfficeForms/hof-template-partials/blob/master/views/confirmation.html

Reply:
err that was initially a mistake

but I've now deleted the confirmation.html file and it won't show my markdown

Reply:
I think I need to call my markdown what-happens-next